src/documents/InfusionEventSystem.md

```diff
@@ -124,12 +135,63 @@ listeners: {
+
+<table>
+  <thead>
+    <tr>
+      <td>Member Name</td>
+      <td>Type</td>
+      <td>Description</td>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>listener</code>, <code>func/funcName</code></td>
+    <td><code>String</code></td>
+    <td>This holds the actual designation of the function which is to be the listener, which was the string that consisted of the entire "short form" for the listener definition above. This, as before, holds either 
+    the name of a global function or an IoC reference to a function. The `func`/`funcName` form is supported for consistency with the syntax for [Invokers](Invokers.md) and the name `listener` will be deprecated for the Infusion 2.0 release. 
+    </td>
+  </tr>
+  <tr>
+    <td><code>args</code> (optional)</td>
+    <td>Any JSON type</td>
+    <td>If supplied, this structure will be used to replace the actual arguments received by the listener, rather than necessarily receiving the exact signature fired by the firer. Any [IoC references](IoCReferences.md) or [expanders](ExpansionOfComponentOptions.md) will be expanded
+    at the time of invoking the listeners, with the additional possibility of referring to the original argument list by using the special context `{arguments}'. Consult [Event Injection and Boiling](EventInjectionAndBoiling.md) for more details of this process.</td>
+  </tr>
+  <tr>
+    <td><code>namespace</code> (optional)</td>
+    <td><code>String</code></td>
+    <td>This holds the same string that would have followed the listener name after a period in the "short form" described above. The namespace can serve three functions - i) to ensure that exactly one listener per namespace is registered with this event firer,
+    ii) to serve as a convenient value by which the listener can be identified for later removal by a procedural call to `removeListener`, and iii) to allow the listener to be targetted by a _priority rule_ (see the next option for details).
+    </td></tr>
+  <tr>
+    <td><code>priority</code> (optional)</td>
+    <td><code>String</code> or <code>Number</code></td>
+    <td>This field allows the configurer to control the sequence in which several listeners to the same event are notified. The recommended form of this field is either `before:otherNamespace` or `after:otherNamespace` where `otherNamespace` represents the `namespace` of some other
+    listener attached to this firer. The framework will sort all the listeners attached to a single firer so that such a listener is fired immediately before or after the other target listener, unless a further constraint positions a third listener in between them.
+    If a group of listeners express a cyclic set of constraints, the framework will signal an error. If there is no listener with the target namespace, the constraint is ignored. There are other possibilities for the `priority` field which are not
```

Comment:
If we don't recommend the other possibilities, perhaps best to just not mention them at all. The link to the Priorities page should be sufficient.